src/maus/reader/flat_ahb_reader.py

```diff
@@ -217,6 +227,20 @@ def _is_segment_group(candidate: Optional[str]) -> bool:
             return False
         return _value_pool_entry_pattern.match(candidate) is not None
 
+    _bedingung_pattern = re.compile(r"\[(?P<key>\d+)\]\s*(?P<text>[^\[\]]+)\s*")  # https://regex101.com/r/hN5x9w/1
+
+    @staticmethod
+    def _extract_bedingungen(candidate: Optional[str]) -> Dict[str, str]:
+        """
+        Checks if the given candidate is a bedingung. If no, returns empty dict.
+        If yes returns a dictionary where the Bedingung keys (e.g. "494") are dictionary keys and the Bedingung +
+        texts (e.g. "Der Wert muss ≤ der zum Erzeugungszeitpunkt sein.") are the dictionary values.
```

Comment:
Was meinst du denn hier noch mit der Bedingung bei "the Bedingung + texts"?

Reply:
https://github.com/Hochfrequenz/mig_ahb_utility_stack/pull/105/commits/1500a86dcc0027718c09b0414b376f141ac38d55 muss sich eingeschlichen haben.